content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,310 @@
+
+    a. Clone the rippled repository: 
+
+        git clone https://github.com/ripple/rippled.git
+
+    b. Access the repository on your local:
+
+        cd rippled
+
+    c. Create a build directory and access it:
+
+        mkdir my_build
+        cd my_build
+
+    d. Build rippled for reporting mode: 
+
+        cmake -G "Unix Makefiles" -Dreporting=ON -DCMAKE_BUILD_TYPE=Debug ..
+
+    e. Run the build using CMake:
+
+        cmake --build . -- -j 4
+
+2. Create a configuration file to run rippled in reporting mode. 
+    Make a copy of the example config file,rippled.cfg, and save it as ripple-reporting-mode.cfg in a location that enables you to run rippled as a non-root user. Access the rippled directory and run:
+
+        mkdir -p $HOME/.config/ripple
+
+        cp cfg/rippled-example.cfg $HOME/.config/ripple/rippled-reporting-mode.cfg
+
+3. Edit rippled-reporting-mode.cfg to set necessary file paths. The user you plan to run rippled as must have write permissions to all of the paths you specify here.
+
+    a. Set the [node_db] path to the location where you want to store the ledger database.
+
+    b. Set the [database_path] to the location where you want to store other database data. (This includes an SQLite database with configuration data, and is typically one level above the [node_db] path field.)
+        
+    c. Set the [debug_logfile] to a path where rippled can write logging information.
+
+Note that these are the only configurations required for rippled to start up successfully. All other configurations are optional and can be tweaked after you have a working server.
+
+4. Edit rippled-reporting-mode.cfg to enable Reporting Mode by updating the [Reporting ] stanza in your rippled.cfg. 
+
+    a. Uncomment the Reporting stanza:
+
+        [reporting]
+        etl_source
+        read_only=0
+
+    b. List the rippled sources (ETL sources) to extract data from:
+        
+        [etl_source]
+        source_grpc_port=50051
+        source_ws_port=6006
+        source_ip=127.0.0.1
+
+5. Configure the databases
+
+    a. Specify the Postgres DB for ledger_tx_tables:
+
+        [ledger_tx_tables]
+        conninfo = postgres://newuser:password@127.0.0.1/reporting
+        use_tx_tables=1
+
+    b. Specify the database for node_db.
+
+    <!-- MULTICODE_BLOCK_START -->
+
+    *NuDB*
+
+        [node_db]
+        type=NuDB
+        path=/home/ubuntu/ripple/
+
+        [ledger_history]
+        1000000
+
+    *Cassandra*
+
+        [node_db]
+        type=Cassandra
+        path=
+
+        [ledger_history]
+        1000000
+
+    <!-- MULTICODE_BLOCK_START -->
+
+6. Modify the configuration for rippled to open up ports
+
+    a. Open the public websocket port:
+
+            [port_ws_admin_local]
+            port = 6006
+            ip = 127.0.0.1
+            admin = 127.0.0.1
+            protocol = ws
+
+
+    b. Open the gRPC port:
```

Comment:
This is on the wrong server.

You don't open the gRPC port on reporting mode (as far as I know). You _definitely_ need to do this on the P2P mode server though.